src/api/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""The example library."""
```

Comment:
This should be modified from the skeleton text (feel free to ignore my suggestion if you have a better idea for what to put here):
```suggestion
"""The Li-PCA API library."""
```